### DISCUSSION TITLE
I am finding error regarding processing of application in student developer pack

### DISCUSSION THREAD

**Discussion by Mohtashim-1** [community/community#119419] [GitHub Education]
### Hi everyone,

Hey Mohtashim-1,Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Apr 14, 2024 for @Mohtashim-1.The image you selected does not appear to contain your school name. Your complete school name must appear in your document, not only the school logo. You may include multiple documents together. If your official document is not in English then you may photograph the original next to an English translation.You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote. Your must configure your browser and operating system to allow location access. You may not use a VPN. If you believe that our information about your school is incomplete or incorrect e.g. missing campus location(s), then please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.
--

Hey Mohtashim-1,

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Apr 14, 2024 for [@Mohtashim-1](https://github.com/Mohtashim-1).

The image you selected does not appear to contain your school name. Your complete school name must appear in your document, not only the school logo. You may include multiple documents together. If your official document is not in English then you may photograph the original next to an English translation.

You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote. Your must configure your browser and operating system to allow location access. You may not use a VPN. If you believe that our information about your school is incomplete or incorrect e.g. missing campus location(s), then please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

![image](https://github.com/community/community/assets/64360314/15db62d2-df47-48cb-975d-9c9e82fe4edc)

**Comment by samus-aran** [comment]
Hi @Mohtashim-1 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for.

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
